src/libAtomVM/nifs.c

```diff
@@ -2993,7 +2989,7 @@ static term nif_erlang_make_fun_3(Context *ctx, int argc, term argv[])
     VALIDATE_VALUE(function_term, term_is_atom);
     VALIDATE_VALUE(arity_term, term_is_integer);
 
```

Comment:
We removed  `memory_ensure_free(ctx, FUNCTION_REFERENCE_SIZE)` from `term_make_function_reference` but we didn't add it back here.
Is this intended?

Reply:
It wasn't, well spotted!